draft-guidiaz-http-head.md

```diff
@@ -0,0 +1,161 @@
+<pre>
+  WIP: WIP-0029
+  Layer: Consensus (hard fork)
+  Title: Support HTTP-HEAD in RADON
+  Authors: Guillermo Díaz <guillermo@otherplane.com>
+  Discussions-To: ..tbd..
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-02-22
+  License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+Data Requests in the Witnet blockchain currently support the aggregation of data extracted from any combination of  HTTP-GET and HTTP-POST retrievals. This proposal specifies the support of HTTP-HEAD data retrievals, enabling data requests to extract metadata about pre-existing content on the Internet. 
+
+## Motivation and rationale
+
+While HTTP-GET and HTTP-POST data retrievals may cover the majority of use cases, other potential applications rather need to retrieve metadata from allegedly existing resource contents on the Internet, instead of relyng on calls to REST/API endpoints.  
+
+For instance, metadata like the existence, age, size, hash or the content type of some given URL, can be extracted by a single HTTP-HEAD retrieval, without actually forcing witnessing nodes to download the whole content that is being referred. 
```

Comment:
For me, this is the main point of allowing HTTP HEAD. We aren't unlocking a new feature but avoiding witnesses downloading unnecessary data. I'd emphasize that in the first paragraph of "Motivation and rationale".